MapboxNavigation/NavigationViewController.swift

```diff
@@ -369,6 +371,10 @@ public class NavigationViewController: NavigationPulleyViewController, RouteMapV
             }
         }
         
+        if route != routeController.routeProgress.route {
```

Comment:
We can, but it should then be moved to the route’s didSet to avoid initializing an excessive RouteProgress.